india_stocks_api/internal/fivepaisa/database/master_contract_db.py

```
@@ -0,0 +1,227 @@
+            logger.error(f"Download failed: {e}")
+            if current_retry >= max_retries:
+                raise
+
+def process_5paisa_csv(path):
+    """
+    Processes the 5Paisa CSV file to fit InstrumentMaster schema.
+    """
+    logger.info("Processing CSV data...")
+    try:
+        df = pd.read_csv(path)
+    except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)